src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js

```diff
@@ -15,11 +19,12 @@ export function YourUploadedCVs({ CVsList }) {
         <div className="upload-cv-botton">
           <Button
             buttonName="Upload new CV"
-            style={{ backgroundColor: 'black' }}
+            style={{ backgroundColor: 'black' }} 
+            onClick = {()=>{setIsShown(true)}}
           />
+          <AddResume isShown={isShown} setIsShown={ setIsShown}/>
```

Comment:
The good strategy however I was thinking about using display of pop-up:none for close button and calling the AddResume component for clicking on AddResume button. As long as your strategy works sounds good. Thank you @KavyaReddy732

Reply:
it's working. but is it in css we should change. i mean display pop-up:none???

Reply:
Do this instead `{isShown && <AddResume setIsShown={ setIsShown}/>}` and remove isShown from AddResume component.

Reply:
it's not working patrikk. i tried to implement. need help